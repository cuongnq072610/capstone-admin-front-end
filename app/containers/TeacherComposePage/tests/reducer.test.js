import { fromJS } from 'immutable';
import studentComposePageReducer from '../reducer';

describe('studentComposePageReducer', () => {
  it('returns the initial state', () => {
    expect(studentComposePageReducer(undefined, {})).toEqual(fromJS({
      isLoading: false,
      isLoadingClose: false,
      ask: {},
      errors: {},
      messageRes: "",
      isLoadingPin: false,
      isLoadingDelete: false,
    }));
  });
});
