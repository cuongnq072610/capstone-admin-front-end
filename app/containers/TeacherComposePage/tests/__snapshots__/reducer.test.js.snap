// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`studentComposePageReducer returns the initial state 1`] = `
Immutable.Map {
  "isLoading": false,
  "isLoadingClose": false,
  "ask": Immutable.Map {},
  "errors": Immutable.Map {},
  "messageRes": "",
  "isLoadingPin": false,
}
`;
